Library/Usage.cs

```diff
@@ -272,7 +272,7 @@ public static RecurlyList<Usage> List(string subscriptionUuid, string subscripti
             return new UsageList(UrlPrefix(subscriptionUuid, subscriptionAddOnCode) + "?" + parameters.ToString());
         }
 
-        public static Usage Get(string subscriptionUuid, string subscriptionAddOnCode, long usageId)
+        public static Usage Get(string subscriptionUuid, string subscriptionAddOnCode, long? usageId)
```

Comment:
Now that this can be `null`, does it need to add guard logic or a fallback? L285 looks like the only place `usageId` is referenced. It's been a while, but unlike say, Ruby, I don't think `null` has a `ToString` method in C#. Either way though -- runtime error or constructing a URL with the literal string`"null"` -- feels like it might be an issue.

Reply:
I'm putting a pause on this for now since the audience affected is minimal and I'd like to do more research on `Id` here: https://github.com/recurly/recurly-client-dotnet/blob/v2/Library/Usage.cs#L19.